about.html

```diff
@@ -0,0 +1,443 @@
+                  <a
+                    href="https://github.com/yourusername"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <img
+                      src="assets/icons/github.svg"
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <footer class="max-w-7xl mx-auto px-8">
+      <nav class="-mb-6 columns-2 sm:flex sm:justify-center sm:space-x-12">
+        <div class="pb-6">
+          <a
+            href="/index.html"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Home</a
+          >
+        </div>
+        <div class="pb-6">
+          <a
+            href="/about.html"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >About</a
+          >
+        </div>
+        <div class="pb-6">
+          <a
+            href="/shop.html"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Shop</a
+          >
+        </div>
+        <div class="pb-6">
+          <a
+            href="/blog.html"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Blog</a
+          >
+        </div>
+        <div class="pb-6">
+          <a
+            href="/contact.html"
+            class="text-lg font-semibold text-travel-orange hover:text-orange-600"
+            >Contact</a
+          >
+        </div>
+      </nav>
+      <div class="mt-10 flex justify-center space-x-10">
+        <a href="https://twitter.com/?lang=en">
+          <span class="sr-only">X</span>
+          <img src="/assets/icons/x.svg" alt="X icon" />
+        </a>
+        <a href="https://en-gb.facebook.com/">
+          <span class="sr-only">Facebook</span>
+          <img src="/assets/icons/facebook.svg" alt="Facebook icon" />
+        </a>
+        <a href="https://uk.linkedin.com/">
+          <span class="sr-only">LinkedIn</span>
+          <img src="/assets/icons/linkedin.svg" alt="LinkedIn icon" />
+        </a>
+        <a href="https://www.pinterest.co.uk/">
+          <span class="sr-only">Pinterest</span>
+          <img src="/assets/icons/pinterest.svg" alt="Pinterest icon" />
+        </a>
+        <a href="https://www.youtube.com/">
+          <span class="sr-only">YouTube</span>
+          <img src="/assets/icons/youtube.svg" alt="Youtube icon" />
+        </a>
+        <a href="https://www.instagram.com/">
+          <span class="sr-only">Instagram</span>
+          <img src="/assets/icons/instagram.svg" alt="Instagram icon" />
+        </a>
+      </div>
+      <p class="mt-10 text-center text-lg text-black">
+        &copy; 2024 TravelUp, Inc. All rights reserved.
+      </p>
+    </footer>
```

Comment:
Styling different than landing page, refer to index.html for changes.  Missing padding all sides.

Reply:
done